superset/Dockerfile

```diff
@@ -1,21 +1,19 @@
 # syntax=docker/dockerfile:1@sha256:ac85f380a63b13dfcefa89046420e1781752bab202122f8f50032edf31be0021
 
-# Get statsd-exporter
-ARG STATSD_EXPORTER
-FROM oci.stackable.tech/sdp/statsd-exporter:${STATSD_EXPORTER} as statsd-exporter
-
 # Build stage
 FROM stackable/image/vector AS builder
 
 ARG PRODUCT
 ARG PYTHON
 ARG AUTHLIB
+ARG STATSD_EXPORTER
 
 COPY superset/constraints-${PRODUCT}.txt /tmp/constraints.txt
 
 RUN microdnf update \
     && microdnf install \
         --assumeyes \
+        tar \
```

Comment:
Please add a comment what it's needed for

Reply:
Done